doc/topics/development/faq-py2-deprecation.rst

```diff
@@ -0,0 +1,115 @@
+
+`SEP
+5 <https://github.com/saltstack/salt-enhancement-proposals/blob/master/accepted/0005-retire-py2-support.md>`__
+was approved in April, 2019.
+
+Can I contribute Python 2 code?
+-------------------------------
+
+For several years, Salt has been a Python 2/3 codebase, requiring Python
+3 support for all contributions. The only changes contributors can
+expect to this process is that we will now accept Python 3-only code
+changes.
+
+While there *is* a significant subset of Python that is compatible with
+both Python 3 and legacy Python, changes requiring ``six`` or otherwise
+removing Python 3-only code will not be accepted.
+
+What if my OS does not include Python 3 packages? Or, how do I upgrade from Salt on Python 2 to Salt on Python 3?
+-----------------------------------------------------------------------------------------------------------------
+
+There are several options to upgrade to Python 3.
+
+On Modern Platforms
+~~~~~~~~~~~~~~~~~~~
+
+At this point, most modern Linux distributions have Python 3 packages.
+Ubuntu 20.04 LTS has moved to Python 3.8 as it’s default system Python.
+
+On Other Platforms
+~~~~~~~~~~~~~~~~~~
+
+Build Your Own
+^^^^^^^^^^^^^^
+
+If your current distribution does not have Python 3 it’s `pretty simple
+to use Salt <https://www.youtube.com/watch?v=Zq0XXtIKx_Q>`__ to build
+and distribute Python 3 on your own. Installing Python 3 and pip
+installing Salt gives you the most control over your distribution.
+
+``pop-build`` distributions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Another approach is to use the ``pop-build`` distribution of Salt. With
+the Sodium release we will begin releasing packages for Salt using
+``pop-build``, in tandem with our normal build process. The
+``pop-build`` distribution will contain Salt, Python, and any necessary
+dependencies, which will make it trivial to have a completely supported
+version of Salt.
+
+Will you support a Python 3 master with an older Python 2 minion?
+-----------------------------------------------------------------
+
+Yes! Our policy of keeping newer masters backwards compatible with *at
+least* the most recent minion version is not changing. You should be
+able to run modern Salt Masters on Python 3 with slightly older minions
+running on legacy Python. Of course, if you want your minions to be able
+to take advantage of the newest features in Salt, upgrading to Python 3
+will be necessary.
+
+Will you support Python 2 master with a new Python 3 minion?
+------------------------------------------------------------
+
+In keeping with our existing policy, we make no guarantees about older
+masters with newer minions. It *may* work, but breakages are common and
+should be expected.
+
+How does this impact Salt-SSH support?
+--------------------------------------
+
+Salt-SSH will need Python 3 on the target minions. We will be upgrading
+Salt-SSH to provide instructions or recommendations on adding Python3 if
+it is not detected.
+
```

Comment:
Minor phrasing suggestion: We are upgrading Salt-SSH to provide instructions...

In general, I recommend avoiding "to be" verb forms.